src/helper/Site_Letsencrypt.php

```diff
@@ -700,52 +700,6 @@ private function getOrCreateDistinguishedName( $domain, array $alternativeNames,
 		return $distinguishedName;
 	}
 
-
```

Comment:
I've deleted this function since it was not being used anywhere within our code, and there was no proper way of defining `$input` and `$output` variables

Reply:
Cool